package.json

```diff
@@ -56,5 +56,8 @@
   "bugs": {
     "url": "https://github.com/aerogear/graphback/issues"
   },
-  "homepage": "https://github.com/aerogear/graphback#readme"
+  "homepage": "https://github.com/aerogear/graphback#readme",
+  "resolutions": {
```

Comment:
Out of curiosity. Why this was required?

Reply:
@wtrocki when I run yarn install from root after cloning, yarn resolved two versions of `@types/react .`  And `@types/react-dom`  also has a dependency to `@types/react`  but it depends on any of the versions of `@types/react` . So yarn tries to resolve 2 versions of `@types/react` and a conflict occurs. So package.json specifies which version of `@types/react` to resolve.
[https://stackoverflow.com/questions/52399839/duplicate-identifier-librarymanagedattributes](stackoverflow)

I tried some other workarounds like deduplicating yarn.lock but nothing worked for me. Anyway it's a issue with my local machine. I was able to build it in my server without resolutions in package.json. So you are correct. I shouldn't have commited that line. Sorry

Reply:
No need to be sorry. I would say it is valid scenario. Create PR with just this change. I will merge it.

Reply:
Just want to clarify something about that pull request. What do you mean by "just this change ?"

1. Do i have to checkout to a new branch, remove resolutions from package.json , commit, push and pull request ? 

2. Or checkout to new branch cherry pick a commit with package.json unedited and without any other commits ? So it's like right after yarn install without the resolution part in package.json.  "Just this change" sounds like this scenario but it doesn't make sense because it's like a PR with fresh built.

Reply:
Please create separate PR and copy paste this 3 lines of code into it. No need for got magic. This PR on the other hand needs some git work. I would squash commits and then redo this single commit to contain only relevant changes

Reply:
Okay just copied 3 lines and created a pull request. What about the changes you requested from yesterday's review ? Shall I create another pull request with all the suggested changes ? So it will be easier for you to merge that PR instead of this ?